feat: adding FlameBloc mixin to allow its usage with enhanced FlameGame classes (#1399)
flame-engine/flame: packages/flame_bloc/README.md

)
```

> To enable the features from `flame_bloc` into your game, you can make your game class inherit from

Reviewer: ```suggestion
To enable the features of `flame_bloc` in your game, you can make your game class inherit from
```

> `FlameBlocGame`, or if you already is using an enhanced `FlameGame` class (like fir example a

Reviewer 2: ```suggestion
`FlameBlocGame`, or if you are already using an enhanced `FlameGame` class (like for example a
```
